server/services/pokemon.js

```diff
@@ -1,9 +1,27 @@
 const Pokemon = require('../models/pokemon');
+const config = require('../config');
+const axios = require('axios');
 
 async function findAll() {
-  return Pokemon.find();
+  return await Pokemon.find().exec();
+}
+
+async function findById(id) {
+  // TODO: Database call
```

Comment:
I assume we are using MongoDB Atlas for this?

Reply:
yessir